src/mbgl/layout/symbol_instance.hpp

```diff
@@ -7,17 +7,27 @@
 
 namespace mbgl {
 
-struct Anchor;
```

Comment:
Is there a story behind the previous use of `struct` and the preference now for `class`?